src/main/scala/nl.knaw.dans.easy.solr4files/components/DDM.scala

```diff
@@ -87,23 +100,14 @@ object DDM {
       .contains("id-type:DOI")
   }
 
-  private val abrMaps = loadVocabularies("https://easy.dans.knaw.nl/schemas/vocab/2012/10/abr-type.xsd")
-    .map { case (k, v) => // attributes in xsd are complex/periode
-      (s"$abrPrefix$k", v) // we want to search with DDM attributes which are abr:ABRcomplex/abr:ABRperiode
-    }
-
-  private val audienceMap = loadVocabularies(
-    "https://easy.dans.knaw.nl/schemas/vocab/2015/narcis-type.xsd"
-  )("Discipline")
-
   private def loadVocabularies(xsdURL: String): Map[String, Map[String, String]] = {
-    val xmlDoc = resource.managed(
-      new URL(xsdURL).openStream()
-    ).acquireAndGet(XML.load) // TODO error handling
-    (xmlDoc \ "simpleType")
-      .map(n => (n.attribute("name").head.text, findKeyValuePairs(n)))
+    for {
+      url <- Try(new URL(xsdURL))
+      xml <- url.loadXml
+    } yield (xml \ "simpleType")
+      .map(n => (n.attribute("name").map(_.text).getOrElse(""), findKeyValuePairs(n)))
       .toMap
-  }
+  }.getOrElse(Map[String, Map[String, String]]())
```

Comment:
replace with `Map.empty`. The compiler already knows about the specific types...